osdep/timer-win2.c

```diff
@@ -66,16 +66,16 @@ void mp_sleep_us(int64_t us)
     mp_end_hires_timers(hrt);
 }
 
-uint64_t mp_raw_time_us(void)
+uint64_t mp_raw_time_ns(void)
 {
     LARGE_INTEGER perf_count;
     QueryPerformanceCounter(&perf_count);
 
     // Convert QPC units (1/perf_freq seconds) to microseconds. This will work
```

Comment:
Just noticed but this should say `nanoseconds` now.

Reply:
fixed